corso_website_springboot/src/test/java/com/corso/springboot/Order_Subdomain/datalayer/OrderRepositoryPersistenceTest.java

```diff
@@ -0,0 +1,37 @@
+package com.corso.springboot.Order_Subdomain.datalayer;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.util.List;
+
+@DataJpaTest
+public class OrderRepositoryPersistenceTest {
+
+    @Autowired
+    OrderRepository orderRepository;
+
+    private Order preSavedOrder;
+
+    @BeforeEach
+    public void setup(){
+        orderRepository.deleteAll();
+        preSavedOrder = orderRepository.save(
+                new Order("982577e6-1909-46b8-8583-e08c9daa4e9b", "2023-01-01", "2023-02-01", Status.IN_PROGRESS, "Custom web development project", "John Doe", "john@example.com", "1234567890", "123 Main St")
+        );
+    }
+
+//    @Test
```

Comment:
Same idea as my previous comment. If this test isn't necessary consider removing it. Good work tho on your persistence test!

Reply:
yep, they will be removed!